content/docs/dvclive/api-reference/live/next_step.md

```diff
@@ -1,7 +1,5 @@
 # Live.next_step()
 
-Signals that the current step has ended and increases `step` value by 1 (one).
```

Comment:
Per the previous comment by @jorgeorpinel, why get rid of this? I find it a useful summary.